.travis.yml

```diff
@@ -1,8 +1,16 @@
+---
 language: bash
 script: true
 sudo: required
-before_install:
-- bash scripts/install.sh
 group: stable
 dist: precise
 os: linux
```

Comment:
I would suggest changing this first bit to:

```yaml
---
language: python
python: 2.7
cache: pip
```

Reply:
We can add the rest to this later.

Reply:
changes made